src/store/selectors/getTopicsBySearchTerm.js

```diff
@@ -0,0 +1,25 @@
+import { includes } from 'lodash'
+import { createSelector as ormCreateSelector } from 'redux-orm'
+import orm from 'store/models'
+import presentTopic from 'store/presenters/presentTopic'
+
+export const getTopicsBySearchTerm = ormCreateSelector(
+  orm,
+  (_, { searchTerm }) => searchTerm,
+  (session, searchTerm) => {
+    if (!searchTerm) return []
+
+    // FIXME: if the user has been browsing multiple groups, this will
```

Comment:
is this new or old?

Reply:
This is note of an existing issue. 

Generally I did a lot to straighten out some topics stuff around the selector and left this note and the others about multiple groups as documentation's of existing issues made more clear during the refactor. I built-in the hooks for multiple group searches but stopped short of figuring out the backend endpoint to write/ammend for
this. This is what I'll create the ticket about.